pkg/config/middleware.go

```diff
@@ -120,6 +128,162 @@ func setupRecoveryMiddleware() *negroni.Recovery {
 	return r
 }
 
+type OidcConfiguation struct {
+	JwkURI string `json:"jwks_uri"` // This is the only field we care about for now.
+}
+
+type OidcJwk struct {
+	JWKeyID     string   `json:"kid"` // JWTs will have a "kid" that will match one of these
```

Comment:
same here for the naming of the fields.

Reply:
maybe we can have similar structure like https://github.com/square/go-jose/blob/master/jwk.go#L41